styles.css

```diff
@@ -1,98 +1,134 @@
 .title {
-       font-family:Roboto;
+    font-family:Roboto;
 }
 
 .wrapper{
-        margin: auto;
-        width: 80%;
+    margin: auto;
+    width: 80%;
 }
 
 .header{
-       display: flex;
-       justify-content: space-between;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 130px;
 }
 
 .logo, .menu {
-             display: inline-block;
+    display: inline-block;
 }
 
 .logo {
-      position: relative;
-      padding: 40px 0;
-      padding-left: 44px;
-      font: bold 24px/31px pt_serifbold, arial, helvetica, sans-serif;
+    position: relative;
+    padding: 40px 0;
+    padding-left: 44px;
+    font: bold 24px/31px pt_serifbold, arial, helvetica, sans-serif;
 }
 
 .logo__circle {
-              position: absolute;
-              left: 0;
-              top: 50%;
-              transform: translateY(-50%);
-              width: 28px;
-              height: 28px;
-              border-radius: 50%;
-              opacity: .7;
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+    opacity: .7;
 }
 
 .logo__circle_color_dark-blue{
-                             background: #0056EC;
+    background: #0056EC;
 }
 
 .logo__circle_color_blue{
-                        left: 8px;
-                        top: 56%;
-                        background: #C8DBFB;
+    left: 8px;
+    top: 56%;
+    background: #C8DBFB;
 }
 
 .menu {
-      display: flex;
-      align-items: center;
+    display: flex;
+    align-items: center;
 }
 
 .menu__item {
-            position: relative;
-            margin-right: 26px;
-            font: 500 18px/21px robotomedium, arial, helvetica, sans-serif;
-            color: #000;
-            text-decoration: none;
-            transition: opacity .5s;
+    position: relative;
+    margin-right: 27px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/IrirniProject/pullRequest?prid=6241594)